mk/uvmt/vsim.mk

```diff
@@ -113,11 +113,15 @@ VLOG_FLAGS    ?= \
 		-suppress 2181 \
 		-suppress 13262 \
 		-suppress vlog-2745 \
+                -suppress vlog-2643 \
+                -suppress vlog-7027 \
+                -suppress vlog-2697 \
```

Comment:
Since this is for vsim, I don't really mind, but there is a mix of spaces and tabs here, so the code is jagged.

Reply:
My bad (did look to see whether the original code was using tabs or spaces).  Pushed in a fix.